corehq/apps/app_manager/models.py

```diff
@@ -3656,10 +3657,17 @@ class Application(ApplicationBase, TranslationMixin, HQMediaMixin):
     cloudcare_enabled = BooleanProperty(default=False)
     translation_strategy = StringProperty(default='select-known',
                                           choices=app_strings.CHOICES.keys())
-    commtrack_enabled = BooleanProperty(default=False)
     commtrack_requisition_mode = StringProperty(choices=CT_REQUISITION_MODES)
     auto_gps_capture = BooleanProperty(default=False)
 
+    @property
+    @memoized
+    def commtrack_enabled(self):
+        if settings.UNIT_TESTING:
+            return False  # override with .tests.util.commtrack_enabled
+        domain_obj = Domain.get_by_name(self.domain) if self.domain else None
+        return domain_obj.commtrack_enabled if domain_obj else False
```

Comment:
I spoke with @dannyroberts about pulling the lookup out to a separate `get_domain_context` function which handles the environment-specific lookup, but it seemed a bit overblown for this one usage.  If there arise more places that would like context from the domain object, that does seem like a good way to go about it.